server/src/main/java/org/opensearch/search/approximate/ApproximatePointRangeQuery.java

```diff
@@ -444,11 +444,12 @@ public boolean canApproximate(SearchContext context) {
         if (context.trackTotalHitsUpTo() == SearchContext.TRACK_TOTAL_HITS_ACCURATE) {
             return false;
         }
+
```

Comment:
Keeping it consistent with `main` https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/search/approximate/ApproximatePointRangeQuery.java to avoid backport conflicts.